core/scripts/local/AdvanceEMP.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * This script advances time in the EMP forward by some specified amount of seconds (or one hour and one second by default).
+ *
+ * `truffle exec ./scripts/local/AdvanceEMPTime --time 10000`
+ * Advances time by 10 seconds
+ */
+const { toBN } = web3.utils;
+const argv = require("minimist")(process.argv.slice(), { string: ["time"] });
+
+// Deployed contract ABI's and addresses we need to fetch.
+const Governor = artifacts.require("Governor");
+const Registry = artifacts.require("Registry");
+const ExpiringMultiParty = artifacts.require("ExpiringMultiParty");
+
+// Contracts we need to interact with.
+let registry;
+let emp;
+
+/** ***************************************************
+ * Main Script
+ /*****************************************************/
```

Comment:
Nothing to change here, but personally I find these comments to be clutter. I know we use them in a lot places so totally fine to leave it in.

Reply:
yeah actually I agree. I like the top line comment that explains the purpose of the script but I don't need to annotate the main script which obviously is the one and only function